lib/bigcommerce/lightstep/tracer.rb

```diff
@@ -58,13 +58,17 @@ def start_span(name, context: nil, start_time: nil, tags: nil)
         self.active_span = span
 
         # run the process
-        result = yield span
-
-        # finish this span if the reporter is initialized
-        span.finish if reporter_initialized?
-
-        # now set back the parent as the active span
-        self.active_span = last_active_span
+        begin
+          result = yield span
+        rescue StandardError
+          span.set_tag('error', true)
```

Comment:
By not re-raising the error here, are you potentially suppressing exceptions that are wrapped in spans?

Reply:
Oops, that's an oversight. Good catch.